Website/plugins/secondaries/gen-secondaries.raku

```diff
@@ -42,9 +42,9 @@ sub ($pp, %processed, %options) {
     my @transfers;
     #| container for tablesearch plugin
     my @routines = [ ['Category', 'Name' , 'Type', 'Where documented'] , ];
-    counter(:items(%definitions.keys), :header('Generating secondaries'));
```

Comment:
What does 'secondaries' mean in this context?

Reply:
@dontlaugh The Raku documentation sources in `Raku/docs/doc` contain the primary sources. Many document classes (typically under the `doc/Type/` directory. Many types have methods with the same name. It was decided that there should be separate web-pages named for methods/roles/operators etc that appear in different primary documents. HTML files are generated from the primary sources, so they are called 'Secondary' sources.

There are three stages in the process. The zeroth stage is when all the primary sources are rendered and headers are parsed to find methods etc in headers. The first stage in gen-secondaries, gathers similar named data chunks, then the second stage when new html pages are generated.  Since there are really very many secondary files, a progress counter is included. It can be turned off with --no-status.